lumicks/pylake/kymotracker/detail/peakfinding.py

```diff
@@ -51,22 +60,23 @@ def find_kymograph_peaks(
 def peak_estimate(data, half_width, thresh):
     """Estimate initial peak locations from data.
 
-    Peaks are detected by dilating the image, and then determining which pixels did not change. These pixels correspond
-    to local maxima. A threshold is then applied to select which ones are relevant.
+    Peaks are detected by dilating the image, and then determining which pixels did not change.
+    These pixels correspond to local maxima. A threshold is then applied to select which ones are
+    relevant.
 
     Parameters
     ----------
     data : array_like
         A 2D image of pixel data.
     half_width : int
-        How much to dilate the image in pixels. This is value should be half of the width we are looking for
-        (rounded upwards). Prior to peak finding, the image is dilated symmetrically. With a half_width of 1 this
-        means turning [0 0 1 0 0] into [0 1 1 1 0] prior to peak-finding.
+        How much to dilate the image in pixels. This is value should be half of the width we are
+        looking for (rounded upwards). Prior to peak finding, the image is dilated symmetrically.
+        With a half_width of 1 this means turning [0 0 1 0 0] into [0 1 1 1 0] prior to
```

Comment:
Hmm. I don't know if that won't make it more confusing, since there's a bunch of transformations involved. They do not provide a half width (but a full one) and they do not provide it in pixels, but in microns (we convert it and that includes some rounding that would have to be explained as well).

Reply:
OK